src/audio/drc/drc_math_generic.c

```diff
@@ -113,7 +113,7 @@ inline int32_t drc_sin_fixed(int32_t x)
 	const int32_t PI_OVER_TWO = Q_CONVERT_FLOAT(1.57079632679489661923f, 30);
```

Comment:
there is a comment above that is wrong, we need to check if any code that depends on this needs it math updated